themes/default/content/blog/announcing-6-0-of-the-pulumi-aws-classic-provider/index.md

```diff
@@ -0,0 +1,64 @@
+
+This blog post walks you through the list of notable changes in the new major version.
+
+<!--more-->
+
+{{% notes type="info" %}}
+Join Pulumi on Thursday, September 7, 2023 for a live workshop on [getting started with infrastructure as code on AWS](/resources/getting-started-with-iac-pulumi-aws/), which will feature the new AWS 6.0 provider.
+{{% /notes %}}
+
+### SDK Size Optimization
+
+The AWS Classic provider contains more than 1,200 resources with over 12,000 properties between them. As a result, it’s expected that the SDK size of the provider has grown to be fairly large. With that in mind, we invested in some key areas to reduce the size of generated files. Depending on the runtime, the SDK is now up to 90% smaller than it used to be in v5 of the provider!
+
+![aws_sdk](aws_sdk.png)
+
+One particular optimization is related to `wafv2` resources, that were particularly difficult to use in Pulumi AWS v5 (issue [one](https://github.com/pulumi/pulumi-aws/issues/2276), [two](https://github.com/pulumi/pulumi-aws/issues/1117[), [three](https://github.com/pulumi/pulumi-aws/issues/2250)). The new release includes a refactored `wafv2` module which uses recursive types. Using recursive types allowed us to remove a lot of excess generated code.
+
+Smaller SDK sizes mean faster downloads, snappier experience with editors and code completion tools, less time needed for `pulumi preview` and `pulumi up` operations, and smaller binaries, all of which contribute to a streamlined developer experience for Pulumi users.
+
+### Terraform Plugin Framework Support
+
+More and more Terraform resources are migrating to the [Terraform Plugin Framework](https://developer.hashicorp.com/terraform/plugin/framework). We are excited to announce that 6.0 supports all resources implemented with both Plugin SDK and Plugin Framework technologies. This means that users get access to the entire API of the upstream provider.
+
+### New Resources and Functions
+
+The new major version of the AWS Classic provider brings 56 new resources and 23 new functions. Additions include resources for Audit Manager, Chime, Fin Space, Global Accelerator, Open Search, Quicksight, Resource Explorer, Route53, Transfer, and many other AWS services.
+
+You can see a full list of new resources and functions in [Release Notes](https://github.com/pulumi/pulumi-aws/releases/tag/v6.0.0).
+
+### Upstream Breaking Changes
+
+Pulumi AWS Classic 6.0 ships all improvements and bug fixes of upstream versions from 5.0.0 to 5.11.0. It also contains a number of upstream breaking changes. Please refer to the [changelog](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md) to navigate the entire list.
+
+### Fixes for Highly Upvoted Issues
+
+The new release ships a few Pulumi-specific fixes as well. A good example is a highly upvoted issue [unexpected behavior of defaultTags](https://github.com/pulumi/pulumi-aws/issues/1655) that has been fixed and is available in the new release.
```

Comment:
Should we list each of these inline here?  I thought there were a few others?

Reply:
As we've resolved 1423, I made an update to the text here.